packages/manager/apps/dedicated/client/app/dedicated/server/dashboard/dashboard.routing.js

```diff
@@ -195,6 +195,19 @@ export default /* @ngInject */ ($stateProvider) => {
         }),
       vrackInfos: /* @ngInject */ ($stateParams, Server) =>
         Server.getVrackInfos($stateParams.productId),
+      ramUpgradable: /* @ngInject */ ($http, $stateParams) =>
+        $http
+          .get(`/order/upgrade/baremetalMemory/${$stateParams.productId}`)
```

Comment:
Please avoid using `$stateParams` especially on the routing part.

Maybe you can rely on the parent state resolve to get access to the `productId`.
see: https://github.com/ovh/manager/blob/master/packages/manager/apps/dedicated/client/app/dedicated/server/server.routing.js#L28

Reply:
i change it, i dont know is a bad practice to use `$stateParams`. :)